mailer.js

```diff
@@ -237,10 +244,70 @@ module.exports = function (log) {
       headers: {
         'X-Link': link
       },
-      subject: gettext('A new device is now syncing to your Firefox Account'),
+      subject: gettext('New sign-in to Sync'),
       template: 'newSyncDeviceEmail',
       templateValues: {
-        resetLink: link
+        resetLink: link,
```

Comment:
I didn't want to stomp on more translation strings than we already have for an "invisible" change.